_includes/proposal.ext

```diff
@@ -0,0 +1,16 @@
+{% comment %}
+Liquid template to display GSoC proposals for a given organization. To be included in a Markdown page.
+Organization is defined by 'organization' attribute in front matter of the parent page.
+Organiization title is defined by 'title' attribute in front matter of the parent page. It defines the proposal
+grouping by organization and is case insensitive.
+{% endcomment %}
+
+{:.table .table-hover .table-striped}
+
+# Project Proposals by {{ page.title }}
+
+{% for proposal in site.gsocproposals %}
+{% capture u_proposal_org %}{{ proposal.organization | upcase }}{% endcapture %}
+{% capture u_org %}{{ page.organization | upcase }}{% endcapture %}
```

Comment:
turns out we want to have multiple organizations per project. Can you look into that?